onnxruntime_extensions/tools/pre_post_processing/docs/pdoc/index.md

```diff
@@ -0,0 +1,371 @@
+Module pdoc
```

Comment:
Seems that we don't need this file.

Reply:
I cleared out the docs directory and generated from scratch. This file does seem pointless, but it's easier to leave the generated output as is so there are less diffs the next time someone updates the docs.